### DISCUSSION TITLE
How to disable Virtual text diagnostics with lsp_lines enabled?

### DISCUSSION THREAD

**Discussion by navxio** [LazyVim/LazyVim#1066] [Q&A] [answered]
I already have this in my config
`vim.diagnostic.config({ virtual_text = false })`
and have disabled `trouble.nvim` using `  { "folke/trouble.nvim", enabled = false },
`
<img width="1116" alt="image" src="https://github.com/LazyVim/LazyVim/assets/9297865/964e2303-3bbc-4ca0-b3a7-cc5425ff366e">

**Comment by dpetka2001** [comment, marked as answer]
There's no `vim.diagnostic.config` in LazyVim. The options for this go to `opts.diagnostic`. See [docs](https://www.lazyvim.org/plugins/lsp#nvim-lspconfig) for how to configure it.
